src/lib.rs

```diff
@@ -337,8 +349,37 @@ pub fn parse_idf(path: &str) -> HashMap<String, Peripheral> {
         );
     }
 
-    // println!("Interrupt information: {:#?}", interrupts);
+    /*
+        Peripherals required by interrupts but not extracted from the source
+    */
+    peripherals.insert("WIFI".to_string(), Peripheral::default());
+    peripherals.insert("RWBT".to_string(), Peripheral::default());
+    peripherals.insert("RWBLE".to_string(), Peripheral::default());
+    peripherals.insert("MMU".to_string(), Peripheral::default());
+    peripherals.insert("MPU".to_string(), Peripheral::default());
+    peripherals.insert("CACHE".to_string(), Peripheral::default());
+    peripherals.insert("SDIO".to_string(), Peripheral::default());
```

Comment:
This is `SDMMC` peripheral (sdmmc_reg.h, sdmmc_struct.h). I think there was an issue generating SVDs for this peripheral, since the register header file does not follow the common format.